contracts/tokenomics/generator/src/contract.rs

```diff
@@ -152,6 +160,7 @@ pub fn execute(
     msg: ExecuteMsg,
 ) -> Result<Response, ContractError> {
     match msg {
+        ExecuteMsg::DeactivatePools { pair_types } => deactivate_pools(deps, env, pair_types),
```

Comment:
info.sender != cfg.owner check should be here

Reply:
In ticket says that anyone can call the Generator contract and reset alloc_points